lib/components/TimezoneDropdown.js

```diff
@@ -45,6 +45,19 @@ var TimezoneDropdown = React.createClass({
     }
   },
 
+  componentDidMount: function() {
+    var self = this;
+    self.updateSuggestedInterval = setInterval(
```

Comment:
Interesting... The trendier cousin of `redux-thunk`, a lib called `redux-saga`, has the advantage of working well for timer-type tasks. I guess we should keep an eye on how often we need such things, although I think it'd take a lot to convince me that we should port to sagas from thunks in the uploader - it would be a fair amount of porting.